packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -0,0 +1,1132 @@
+{
```

Comment:
Would it make sense to sort these keys in the end?

Reply:
Yes, I'll organize them in a way that makes sense once I'm done adding to it.